app/ui/src/main/java/com/fsck/k9/ui/managefolders/FolderSettingsViewModel.kt

```diff
@@ -62,16 +69,36 @@ class FolderSettingsViewModel(
     }
 
     private fun createFolderObject(account: Account, folder: Folder): Folder {
-        val folderType = FolderInfoHolder.getFolderType(account, folder.serverId)
+        folderAccount = account
```

Comment:
The assignments have nothing to do with creating the `Folder` object. They should be moved outside of this method.